docker/truffle/Dockerfile

```diff
@@ -2,11 +2,4 @@ FROM node:10.15-alpine
 
 # Create app directory
 WORKDIR /app/dex-contracts
-# Don't copy git artifacts as they contain backpointers to the parent repo
-COPY dex-contracts/[^.git]* ./
```

Comment:
If you don't copy any dex-contracts files at all, how can it possibly work? Should then just "nothing migrate"?
Could you explain what you are doing? sorry, I don't see it right now.

Reply:
Check out 

https://github.com/gnosis/dex-services/blob/3268c7faca3f800b2c66bdf477ce854141239e77/docker-compose.yml#L36-L44

The contracts are included as a volume, thus they are taken from the local file system at runtime. When at runtime the run script is executed the most recent host contracts are used.

This means npm install/network-inject etc. can happen on the host, but you can still get your entire environment in a demo/test ready state by running `docker-compose up` (instead of `docker-compose up stablex` + doing migration in another shell)